File: c/claude/p21.c
// Task p21: Read value from command line, add 1000, print result, return 0
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[])
{
    if (argc > 1)
    {
        int value = atoi(argv[1]);
        int result = value + 1000;
        printf("%d\n", result);
    }
    else
    {
        printf("Usage: %s <number>\n", argv[0]);
    }

    return 0;
}

File: python/claude/p21.py
# Task p21: Read value from command line, add 1000, print result, return 0
import sys

if len(sys.argv) > 1:
    try:
        value = int(sys.argv[1])
        result = value + 1000
        print(result)
    except ValueError:
        print("Invalid input.")
else:
    print("Usage: python p21.py <number>")
